### PR TITLE
Updated grades

### PR DESCRIPTION
**Pull request by smoga-git**

### CODE REVIEW

File: src/main/resources/data.sql [hunk 1]
```diff
@@ -1,14 +1,18 @@
...
 	('AO', 'Administrative Officer',null),
     ('EO', 'Executive Office',null),
-	('SEO','Senior Executive Office',null),
-    ('HEO','Higher Executive manager',null),
- 	('G6', 'Grade 6', null),
-  	('G7', 'Grade 7', null),
-	('SCS', 'Director', null);
+   	('G6', 'Grade 6', null),
+    ('G7', 'Grade 7', null),
+    ('HEO','Higher Executive manager', null),
```
Comment (m24murray) [inline src/main/resources/data.sql]: Should this be `Higher Executive Officer` as per the ticket?

File: src/main/resources/data.sql [hunk 2]
```diff
@@ -1,14 +1,18 @@
 SET FOREIGN_KEY_CHECKS = 0;
 
 INSERT INTO grade (code, name, organisation_id) VALUES
-	('AA', 'Administrative level',null),
+	('AA', 'Administrative Assistant',null),
 	('AO', 'Administrative Officer',null),
     ('EO', 'Executive Office',null),
```
Comment (m24murray) [inline src/main/resources/data.sql]: Should this be `Executive Officer` as per ticket?

File: src/main/resources/data.sql [hunk 3]
```diff
@@ -1,14 +1,18 @@
...
-	('SEO','Senior Executive Office',null),
-    ('HEO','Higher Executive manager',null),
- 	('G6', 'Grade 6', null),
-  	('G7', 'Grade 7', null),
-	('SCS', 'Director', null);
+   	('G6', 'Grade 6', null),
+    ('G7', 'Grade 7', null),
+    ('HEO','Higher Executive manager', null),
+    ('SEO','Senior Executive Office', null),
+    ('PB1', 'Senior Civil Service - Deputy Director', null),
```
Comment (m24murray) [inline src/main/resources/data.sql]: Can you check whether `PB1` is sufficient or should we have `PB1/1A`?

File: src/main/resources/data.sql (hunk 1 above)
Comment (smoga-git) [reply src/main/resources/data.sql]: yup, changed

**Pull request merged**